boris-http/src/Boris/Http/Route.hs

```diff
@@ -407,16 +400,30 @@ route store authentication buildx logx projectx mode = do
       log'' <- liftError Store.renderFetchError $
         Build.logOf store logx (BuildId buildId)
 
-      case log'' of
-        Nothing -> do
+      withAccept $ \case
+        AcceptHTML -> do
           Spock.setStatus HTTP.notFound404
           Spock.html "TODO: 404 page."
-        Just logs ->
-          Spock.stream $ \send flush ->
-            runConduit $
-              logs =$=
-                CL.mapM_ (\t -> send (fromByteString t) >> flush)
-
+--          case log'' of
+--            Nothing -> do
+--              Spock.setStatus HTTP.notFound404
+--              Spock.html "TODO: 404 page."
+--            Just b ->
+--              View.render $ View.build b
+        AcceptJSON -> do
+          case log'' of
+            Nothing -> do
+              Spock.setStatus HTTP.notFound404
+              Spock.json ()
+            Just logs -> do
```

Comment:
`logs` has type `Source IO ByteString`

Reply:
But now, by using `logOf'` that returns `[Text]`, things become much simpler.

Reply:
I think this is the place to make a decision on what you want to be dealing with. The options would be:

 - `stream` the actual text log.
 - `json` a data-structure containing log lines (it is fine for it to be json of a big string for the moment, but longer term it makes more sense for it to have the timestamps etc... so client can do interesting things.

And it should be independent of whether the log was stored in cloudwatch or db. 

Looking at everything, the `json` one looks like the right call for now. Which just means `logOf'` isn't complete, and if it was you would be done. I will add another comment back there about how it could work.